test/src/e2e_vm_tests/test_programs/should_fail/enum_rust_like/test.toml

```diff
@@ -3,9 +3,9 @@ category = "fail"
 # check: $()error
 # check: enum_rust_like/src/main.sw:7:5
 # check: $()Ok,    // Illegal
-# nextln: ^ Expected ':'. Enum variants must be on the form 'Foo : ()' or 'Bar : u32'.
+# nextln: $()^ Expected ':'. Enum variants must be on the form 'Foo : ()' or 'Bar : u32'.
```

Comment:
There is no need to write the leading `^ `.

Reply:
Fixed.